OJS.Workers.SubmissionProcessors/ExecutionTypeFilters/SubmissionFilteringServiceBase.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace OJS.Workers.SubmissionProcessors.ExecutionTypeFilters
+{
+    using System.Collections.Generic;
+    using OJS.Workers.Common;
+    using OJS.Workers.Common.Models;
+    using OJS.Workers.SubmissionProcessors.Workers;
+
+    public abstract class SubmissionFilteringServiceBase
+            : ISubmissionsFilteringService
+    {
+        protected abstract ISet<ExecutionStrategyType> EnabledExecutionStrategyTypes { get; }
+
+        protected abstract ISet<ExecutionStrategyType> DisabledExecutionStrategyTypes { get; }
+
+        public bool CanProcessSubmission(IOjsSubmission submission, ISubmissionWorker submissionWorker)
+            => submission != null
+                && !this.DisabledExecutionStrategyTypes.Contains(submission.ExecutionStrategyType)
+                && this.EnabledExecutionStrategyTypes.Contains(submission.ExecutionStrategyType)
```

Comment:
Isn't easier to have just disabled execution strategies and all others will be enabled by default. That way remote and local workers will define only their disabled strategies.

Reply:
I aggre, in the case, where one worker supports all strategies. But, currently, if I have to list all supported or unsupported strategies ... this will be a very big list...

Reply:
Only the unsupported. For LocalWorker will be 3, for the other they are already listed, but commented out.